guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -10,13 +10,21 @@ This information allows Katello to display manifest labels, annotations, and inf
 As a result, migrations must be performed to pull this content from manifests into the database.
 
 ifdef::katello[]
-This migration takes time, so if you depend on container content and need minimal upgrade downtime, run the following command on {ProjectServer} for a pre-migration:
+.Procedure
+. This migration takes time, so if you depend on container content and need minimal upgrade downtime, enter the following commands in a `tmux` window on {ProjectServer} for a pre-migration:
++
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} advanced procedure run pulpcore-container-handle-image-metadata
+----
+This command migrates data while {Project} is running without any need for downtime and reduces future upgrade downtime.
 
+. To allow the container image API to display manifest labels, annotations, and if the manifest represents bootable or flatpak content, enter the following command:
```

Comment:
You can drop "enter the command".

But more importantly, is "if the manifest represents [...]" a part of a decision the user must make? Is this a command that users only run when this condition is true?

If that's the case, you should probably lead with that: "If the manifest represents [...], allow the container image [...]".